internal/builders/bazel/rebuilder.sh

```diff
@@ -0,0 +1,490 @@
+#!/bin/bash
+#
+# Copyright 2023 SLSA Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: -u not set to check for empty variables from parse arguments function.
+set -eo pipefail
```

Comment:
you can still set `-u`. The. you can check for empty variables by using `${VAR:-}`

Reply:
thanks for the tip! set -u and used ${VAR:-}